Library/Formula/truncate.rb

```diff
@@ -0,0 +1,21 @@
+class Truncate < Formula
+  desc "This program truncates a file to a given size"
+  homepage "http://www.vanheusden.com/truncate"
+  url "http://www.vanheusden.com/truncate/truncate-0.8.tgz"
+  sha256 "165914d643351324d69096427d3ce5d3ab13aed3f7c990a71ba3bd0c6f278f00"
+
+  def install
+    inreplace "truncate.c" do |s|
+      s.gsub! '#include "error.h"', ""
+      s.gsub! "off64_t", "off_t"
+      s.gsub! "stat64", "stat"
+      s.gsub! "ftruncate64", "ftruncate"
+    end
```

Comment:
error.h does not come with the program, but it compiles without it fine
OSX off_t, stat and ftruncate are already natively 64bit and these *64 are either missing or deprecated in 10.11.
I made a patch at first, but it is quite large, since there is a lot of these calls. So I though the less code in formula the better